package.json

```diff
@@ -14,6 +14,7 @@
     "@aws-sdk/client-api-gateway": "3.670.0",
     "@aws-sdk/client-apigatewaymanagementapi": "3.670.0",
     "@aws-sdk/client-apigatewayv2": "3.670.0",
+		"@aws-sdk/client-batch": "3.662.0",
```

Comment:
Total nitpick, but can we fix the spacing?

Reply:
Already did